analyzer/block/block.go

```diff
@@ -156,6 +165,72 @@ func (b *blockBasedAnalyzer) fetchBatchForProcessing(ctx context.Context, from u
 	return heights, nil
 }
 
+// Returns info about already-processed blocks in the analyzer's configured range:
+// - Whether the already-processed blocks form a contiguous range starting at "to".
+// - Whether any of the blocks were processed by fast-sync.
+// - The height of the highest already-processed block.
+func (b *blockBasedAnalyzer) processedSubrangeInfo(ctx context.Context) (bool, int64, error) {
+	var isContiguous bool
+	var maxProcessedHeight int64
+	if err := b.target.QueryRow(
+		ctx,
+		queries.ProcessedSubrangeInfo,
+		b.analyzerName,
+		b.blockRange.From,
+		b.blockRange.To,
+	).Scan(&isContiguous, &maxProcessedHeight); err != nil {
+		return false, 0, err
+	}
+	return isContiguous, maxProcessedHeight, nil
+}
+
+// Returns true if the block immediately preceding `height` has been processed in slow-sync mode.
```

Comment:
```suggestion
// Returns true if the block at `height` has been processed in slow-sync mode.
```

Or am i misunderstanding the code?